src/Tizen.NUI/src/public/BaseComponents/ViewBindableProperty.cs

```diff
@@ -1086,12 +1086,13 @@ public partial class View
         });
         /// This will be public opened in tizen_5.0 after ACR done. Before ACR, need to be hidden as inhouse API.
```

Comment:
it is internal api. please remove the comment.

Reply:
Hello, I removed the comments.